import io.netty.resolver.dns.DnsResolveContext.DnsResolveContextException;
public class FunctionUse16430 {
public void funcUse() {
DnsResolveContextException dnsresolvecontextexception = new DnsResolveContextException();
dnsresolvecontextexception.newStatic(null,null,null);
}
}